scripts/exregional_run_analysis.sh

```diff
@@ -177,10 +188,15 @@ else
   BKTYPE=1              # cold start
 fi
 
+if [ ${if_observer} == "True" ]; then
+  bkpath=${cycle_dir}/mem1/fcst_fv3lam/RESTART
```

Comment:
mem1 or mem01? 

Also, I think the background should have been prepared by the the "prepstart" task. So here bkpath should be ..../INPUT.